Remove files that were generated by accident

lidimayra opened the pull request:
See https://github.com/artsy/rosalind/pull/789#discussion_r1578214157 for reference

cc @artsy/amber-devs

damassi reviewed (approved):
Ty! Would you mind adding [`noEmit: true`](https://github.com/artsy/volt/blob/main/tsconfig.json#L13) to the tsconfig.json before merging? Then this won't happen again.

lidimayra commented:
> Ty! Would you mind adding [noEmit: true](https://github.com/artsy/volt/blob/main/tsconfig.json#L13) to the tsconfig.json before merging? Then this won't happen again.

Pro move! ⭐  Thanks for the suggestion, just added this and squashed into the original commit!
